spec/services/FetchMoviesSpec.php

```diff
@@ -33,19 +32,16 @@
             expect($movieData->release_date)
             ->toBeA('string');
             expect($movieData->poster_image)
-            ->toBeA('null');
+            ->toBeA('string');
             expect($movieData->critic_rating)
             ->toBeA('double');
             expect($movieData->description)
             ->toBeA('string');
         });
 
-        it('creates an array of Movie Models', function () {
-        });
-
         it("gets the latest movie", function () {
             $fetcher = new FetchMovies();
-            $fetcher->getMovies();
+            $fetcher->getLatestMovie();
             expect($fetcher)->toBeAnInstanceOf(FetchMovies::class);
```

Comment:
here it looks like you are testing that php properly creates instances rather that what you said `gets the latest movie`

Reply:
So I should test for just the return of an instance of the movie class correct?